### ISSUE TITLE
JG: Comments related to introduction

### ISSUE THREAD

**Issue by hturner** [rjournal/rjtools#83]
Comments from @ajrgodfrey that might be incorporated into the introduction

> - Consider mentioning the [W3C Accessibility Maturity Model](https://w3c.github.io/maturity-model/) as a model for continuous improvement of digital accessibility.
> - As the R Journal is published by the R Foundation, it could be considered to subject to Austria's accessibility legislation which may imply certain standards should be met for the website.

**Comment by hturner** [comment]
put on wrong repo
